tests/receivers/sshcheck/testdata/resource_metrics/sshcheck.yaml

```
@@ -0,0 +1,22 @@
+resource_metrics:
+  - attributes: {}
+    scope_metrics:
+      - instrumentation_scope:
+          attributes: {}
+          name: otelcol/sshcheckreceiver
+          version: <ANY>
+        metrics:
+          - description: Measures the duration of SSH connection.
+            name: sshcheck.duration
+            type: IntGauge
+            unit: ms
+          - attributes:
+              error.message: "ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain"
+            description: Records errors occurring during SSH check.
+            name: sshcheck.error
+            type: IntNonmonotonicCumulativeSum
+            unit: '{error}'
```

```suggestion
            unit: "{error}"
```
Just for consistency, this suggestion would match contrib's metadata.yaml value here.